CHANGES.md

```diff
@@ -1,3 +1,23 @@
+### Unreleased
+
+#### Fixed
+- **irmin-pack**
+  - Bug fix in inode, irmin-pack's tree-like data structure speeding up
```

Comment:
I think that we only add small explanations in the changelog. I would only say that "it fixes a bug that lead to an inconsistent hash when removing values from an inode larger than Conf.stable_hash", or something similar, and add more explanations in the commit message?

Reply:
no we should keep the CHANGELOG and/or be even more explicit on the impact: e.g. no external facing changes if the number of entries is smaller than `max_entries`, otherwise `remove`s can lead to non-deterministic hashes. This is very important to be explicit about this kind of impact to our user to point them why they need to upgrade.

Reply:
I've reworded the entry in the changelog. What do you thing?